src/app/authoring-tool/edit-component-advanced/edit-component-advanced.component.html

```diff
@@ -0,0 +1,107 @@
+<h1 mat-dialog-title i18n>Advanced Settings</h1>
+<div mat-dialog-content>
```

Comment:
Make this similar to preview-component-dialog.component.html?
```suggestion
<h1 mat-dialog-title i18n>Advanced Settings</h1>
<mat-divider></mat-divider>
<div mat-dialog-content>
```